blackboxopt/optimizers/botorch_utils.py

```diff
@@ -232,7 +232,8 @@ def posterior_mean(x):
             jac=False,
             x0=x,
             args=(),
-            bounds=bounds,
+            # The numerical representation always lives on the unit hypercube
+            bounds=torch.Tensor([[0, 1]] * len(search_space)).to(dtype=torch_dtype),
```

Comment:
According to the [pytorch docs](https://pytorch.org/docs/stable/tensors.html#torch.Tensor), we could also use
`bounds=torch.tensor([[0, 1]] *len(search_space), dtype=torch_dtype)`, which is the listed way of creating a tensor with pre-existing data.

Reply:
We've seen pylint struggle with the `torch.tensor()` interface, claiming "Module 'torch' has no 'tensor' member" but I haven't figured out why. To keep the noise down in case someone is using pylint extensions in their IDE, I'd stick with the current proposal, assuming that the overhead is negligible here.